README.md

```diff
@@ -1,59 +1,71 @@
 # Git Workshop
+
 ## Brought to you by [GRWebDev](https://www.meetup.com/grwebdev/) Powered by [LaFleur Marketing](https://lafleur.marketing/)
 
 ### Welcome!
-At this workshop you'll learn the basics of [Git](https://git-scm.com/), a version control system.  This technology is the basis for numerous popular platforms, including [GitHub](https://github.com/).
+
+At this workshop you'll learn the basics of [Git](https://git-scm.com/), a version control system. This technology is the basis for numerous popular platforms, including [GitHub](https://github.com/).
```

Comment:
Sooooooo... folks don't appreciate my old school "two spaces after a period or colon"?  Come on; the web automatically gets rid of these anyway, right?  😆

[https://www.grammarly.com/blog/spaces-after-period/](https://www.grammarly.com/blog/spaces-after-period/)

I mean, check out those examples.  Don't they look better with two spaces?  😉

Reply:
@tnamdevnote (I'm being silly of course.  A single space after a period is fine.  👍)

Reply:
Wait, I didn't know that two spaces after a period was a proper punctuation. It's no surprise that two spaces reads better than single spaces in other docs. I guess prettier isn’t as respectful of traditions like kids these days (definitely not me). 🤣

Reply:
@tnamdevnote Well, two spaces was proper back... ummmmm... many decades ago.  😅  Per Grammarly, this isn't a thing anymore, and the web doesn't render multiple spaces without actual non-breaking spaces (`&nbsp;`).  But decades of habits are hard to break, and I get yelled at all the time for my double-spacing habits.  😉

Reply:
@tnamdevnote Check out this Reddit post, and then all of the comments.  HILARIOUS.  😆

[https://www.reddit.com/r/GenX/comments/tbf0ai/fellow_genxers_stop_double_spacing_after_periods/](https://www.reddit.com/r/GenX/comments/tbf0ai/fellow_genxers_stop_double_spacing_after_periods/)

Reply:
My favorite one. 😂 I cannot imagine a world without Oxford comma... 

![image](https://github.com/slothwerks-studio/git-workshop/assets/44216709/5e0c39d4-2c20-4b78-8b73-447e95e0bb5b)